examples/qiskit/0_Getting_Started.ipynb

```diff
@@ -0,0 +1,620 @@
+{
```

Comment:
<p>We don't really explain how qiskit jobs relate to braket tasks. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/265/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
<p>+1, especially since an AWS task is a Qiskit job, and AWS jobs are distinct from tasks</p>

Reply:
Addressed.